src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -491,6 +491,7 @@ public string TooltipText
         /// If all items in the container set this property, their sizes will be proportional to the specified flex factor.<br />
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Deprecated in API8, Will be removed in API11, ")]
```

Comment:
It shouldn't end with comma. There is no suggestion for an alternative property after deprecation, but it is acceptable.
```suggestion
        [Obsolete("Deprecated in API8, Will be removed in API11.")]
```

Reply:
Removed the comma and changed as a single line.